coffee-chats/src/main/java/com/google/step/coffee/APIUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.google.step.coffee;
+
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+/** Utility class holding common tools for Google API usage and authorisation. */
+public class APIUtils {
+
+  public static final String APPLICATION_NAME = "Coffee Chats";
+  public static final HttpTransport HTTP_TRANSPORT = new UrlFetchTransport();
+  public static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
```

Comment:
Feel free to ignore, but it seems weird to use Gson in one part of the app and to use Jackson in another, since they both do pretty much the same thing.

Reply:
Jackson was recommended as the default Factory to be used when interacting with Google's authorisation server, and with the other APIs.